### PR TITLE
OStatus: Fix PubSubHubBub subscription

### PR DESCRIPTION
**Pull request by annando**
It seems as if incoming subscriptions via PubSubHubBub hadn't worked anymore. This should fix it. (I already had got two successful subscriptions after the fix was applied)

### CODE REVIEW

File: src/Module/OStatus/Salmon.php [hunk 1]
```diff
@@ -62,9 +62,10 @@ public function __construct(Database $database, L10n $l10n, App\BaseURL $baseUrl
 	 * @throws HTTPException\OKException
 	 * @throws \ImagickException
 	 */
-	protected function rawContent(array $request = [])
+	protected function post(array $request = [])
 	{
 		$xml = Network::postdata();
+		$this->logger->debug('Gut request data.', ['request' => $request]);
```
Comment (nupplaphil) [inline src/Module/OStatus/Salmon.php]:
```suggestion
		$this->logger->debug('Got request data.', ['request' => $request]);
```

File: src/Module/OStatus/PubSub.php [hunk 2]
```diff
@@ -58,6 +58,7 @@ protected function post(array $request = [])
 
 		$this->logger->info('Feed arrived.', ['from' =>  $this->request->getRemoteAddress(), 'for' => $this->args->getCommand(), 'user-agent' => $this->server['HTTP_USER_AGENT']]);
 		$this->logger->debug('Data stream.', ['xml' => $xml]);
+		$this->logger->debug('Gut request data.', ['request' => $request]);
```
Comment (nupplaphil) [inline src/Module/OStatus/PubSub.php]:
```suggestion
		$this->logger->debug('Got request data.', ['request' => $request]);
```

File: src/Module/OStatus/PubSubHubBub.php [hunk 3]
```diff
@@ -71,6 +71,8 @@ protected function post(array $request = [])
 			throw new HTTPException\ForbiddenException();
 		}
 
+		$this->logger->debug('Gut request data.', ['request' => $request]);
```
Comment (nupplaphil) [inline src/Module/OStatus/PubSubHubBub.php]:
```suggestion
		$this->logger->debug('Got request data.', ['request' => $request]);
```

**Review by nupplaphil — CHANGES_REQUESTED** [review]
Gute Anfragen :-)

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**